RealmTasks Android/app/src/main/java/io/realm/realmtasks/list/TouchHelper.java

```diff
@@ -363,16 +371,24 @@ private void selectView(ItemViewHolder selected, @ActionState int actionState) {
             if (previousActionState == ACTION_STATE_SWIPE) {
                 if (TouchHelper.this.selected != null) {
                     final float maxNiche = logicalDensity * 66 / 2;
```

Comment:
I thought `logicalDensity * 66 / 2` was better than `logicalDensity * 66`, but I will align this with iOS version and your feedback.